src/Tabs/Panel.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { useTabsContext } from './TabsContext';
+import { StyledPanel } from './Panel.styles';
+
+export interface PanelProps {
+  whenActive: string;
```

Comment:
Can this be isActive or something?

Reply:
whenActive makes more sense in this case as it is based on the tab that is active. That's why I have changed it up for the style prop.

Reply:
Can you rename whenActive to be tabId